riotfile.py

```diff
@@ -319,13 +320,16 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         Venv(
             name="telemetry",
             command="pytest {cmdargs} tests/telemetry/",
-            pys=select_pys(),
+            # otel doesn't support python 2.7 and has changes in 3.5
```

Comment:
otel does not support `python<3.6` but instrumentation telemetry should support and test all python versions. 

Also why do we need to import the opentelemetry package in instrumentation telemetry tests? Instrumentation telemetry should be independent.

Reply:
We're importing the otel package to generate otel traces to test span creation metrics.

Reply:
Ah sorry this got added from another PR